test/doc/developer/reverse.cpp

```diff
@@ -0,0 +1,194 @@
+
+#include <eve/wide.hpp>
+#include <eve/function/bit_cast.hpp>
+#include <eve/function/store.hpp>
+
+#include <algorithm>
+#include <bit>
+#include <concepts>
+
+#include <type_traits>
+
+#if defined(EVE_HW_X86)
+
+namespace eve_extra {
+namespace _reverse {
+
+template <typename T, typename N, typename ABI>
```

Comment:
If you know your ABI is x86 related, you should use the `x86_abi` concept instead of `typename`

Reply:
OK